tests/test_agent_runner_streamed.py

```diff
@@ -241,6 +241,7 @@ def remove_new_items(handoff_input_data: HandoffInputData) -> HandoffInputData:
         input_history=handoff_input_data.input_history,
         pre_handoff_items=(),
         new_items=(),
+        run_context=handoff_input_data.run_context,
```

Comment:
I added run_context param in tests but the ones under examples do not have the param. this means backward compatibility is kept